src/Screens/Matzal/Matzal.tsx

```diff
@@ -147,6 +191,24 @@ export const Matzal = () => {
     );
   };
 
+  const cadetCompare = (cadet1: User, cadet2: User) => {
+    // Order unmarked first
+    if (cadet1.attendance.inAttendance !== null
```

Comment:
u can move cadet1.attendance.inAttendance  and cadet2.attendance.inAttendance  to variables